python/tank/authentication/errors.py

```diff
@@ -41,6 +41,21 @@ def __init__(self, msg):
         )
 
 
+class ConsoleLoginWithSSONotSupportedError(ShotgunAuthenticationError):
```

Comment:
i think it's probably worth having an SSO error class that this derives from?

Reply:
Good idea, I will create one.